sql/7_0_1-to-7_0_2_upgrade.sql

```diff
@@ -220,11 +220,11 @@ INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`, `is_default`
 -- Create new list Default Open Tabs
 INSERT INTO `list_options` (`list_id`, `option_id`, `title`) VALUES ('lists', 'default_open_tabs', 'Default Open Tabs');
 -- Populate list with sensible defaults based on previous optiosn in globals, assume no tabs are actually active
-INSERT INTO `list_options` (`list_id`, `notes`, `title`, `seq`, `option_id`, `activity`) VALUES ('default_open_tabs', 'interface/main/main_info.php', 'Calendar', 10, 'cal', '0');
+INSERT INTO `list_options` (`list_id`, `notes`, `title`, `seq`, `option_id`, `activity`) VALUES ('default_open_tabs', 'interface/main/main_info.php', 'Calendar', 10, 'cal', '1');
 INSERT INTO `list_options` (`list_id`, `notes`, `title`, `seq`, `option_id`, `activity`) VALUES ('default_open_tabs', 'interface/new/new.php', 'Patient Search / Add', 20, 'pat', '0');
 INSERT INTO `list_options` (`list_id`, `notes`, `title`, `seq`, `option_id`, `activity`) VALUES ('default_open_tabs', 'interface/main/finder/dynamic_finder.php', 'Patient Finder', 30, 'fin', '0');
 INSERT INTO `list_options` (`list_id`, `notes`, `title`, `seq`, `option_id`, `activity`) VALUES ('default_open_tabs', 'interface/patient_tracker/patient_tracker.php?skip_timeout_reset=1', 'Flow Board', 40, 'flb', '0');
-INSERT INTO `list_options` (`list_id`, `notes`, `title`, `seq`, `option_id`, `activity`) VALUES ('default_open_tabs', 'interface/main/messages/messages.php?form_active=1', 'Message Inbox', 50, 'msg', '0');
+INSERT INTO `list_options` (`list_id`, `notes`, `title`, `seq`, `option_id`, `activity`) VALUES ('default_open_tabs', 'interface/main/messages/messages.php?form_active=1', 'Message Inbox', 50, 'msg', '1');
 -- Activate the 2 list options that were the previous default and second pane settings
 UPDATE `list_options` lo INNER JOIN globals g ON lo.notes = g.gl_value SET lo.activity = 1;
```

Comment:
The above fixes should not be needed if the line above is working as intended. @robertdown , was the above line working on testing?

Reply:
i wonder if instead need (not sure if the % will work here though or of can use LIKE) since i think the originals had stuff before the interface:
```
UPDATE `list_options` lo INNER JOIN globals g ON lo.notes LIKE %g.gl_value SET lo.activity = 1;
```

Reply:
Hopefully following should work:
```
UPDATE `list_options` lo INNER JOIN globals g ON lo.notes LIKE '%g.gl_value' SET lo.activity = 1;
```
or if that does not work then:
```
UPDATE `list_options` lo INNER JOIN globals g ON lo.notes LIKE concat( '%', g.gl_value) SET lo.activity = 1;
```
reference: https://stackoverflow.com/questions/37269803/mysql-join-on-like-statement

Reply:
noticed wasn't working on the dev demos, thanks will try that

Reply:
We should also make this more specific so it doesn't do any odd stuff with other unrelated list options:
`lo.list_id = 'recent_patient_columns'`
`g.gl_name = 'default_top_pane' OR g.gl_name = 'default_second_tab' `

Reply:
would try to add above as a WHERE at end of query:
```
WHERE lo.list_id = 'recent_patient_columns' AND (g.gl_name = 'default_top_pane' OR g.gl_name = 'default_second_tab')
```

Reply:
ok, these 2 lines are testing well (note need the second line for when upgrading in a version before the default_second_tab was added, which for example will impact the official demos):
```
UPDATE `list_options` lo INNER JOIN globals g ON lo.notes LIKE CONCAT('%', g.gl_value) AND lo.list_id = 'default_open_tabs' SET lo.activity = 1 WHERE g.gl_name = 'default_top_pane' OR g.gl_name = 'default_second_tab';
UPDATE `list_options` SET `activity` = 1 WHERE `list_id` = 'default_open_tabs' AND `option_id` = 'msg' AND NOT EXISTS (SELECT `gl_value` FROM `globals` WHERE `gl_name` = 'default_second_tab');
```